src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/features/form/containers/GroupContainer.tsx

```diff
@@ -177,7 +178,7 @@ export function GroupContainer({
   const hidden: boolean = useSelector((state: IRuntimeState) => GetHiddenSelector(state, { id }));
   const formData: IFormData = useSelector((state: IRuntimeState) => state.formData.formData);
   const layout: ILayout = useSelector((state: IRuntimeState) => state.formLayout.layouts[state.formLayout.uiConfig.currentView]);
-  const [editIndex, setEditIndex] = React.useState<number>(-1);
+  const editIndex: number = useSelector((state: IRuntimeState) => state.formLayout.uiConfig.repeatingGroups[id].editIndex);
```

Comment:
Are we sure this is always initialized with -1, or should we default to that here in case? Since this is an optional parameter.

Reply:
Don't think it actually matters since we check for greater than 0, so undefined/null would also work. But added default to -1 just for clarity 👍